src/App.tsx

```diff
@@ -224,9 +224,12 @@ function App() {
   const [maxCells, setMaxCells] = createSignal(200);
   const [drawMode, setDrawMode] = createSignal("");
   const [cellUnionText, setCellUnionText] = createSignal("");
```

Comment:
can we get away with a single textbox that detects if the input starts with `{` for GeoJSON?